packages/yoastseo/src/languageProcessing/researches/wordCountInText.js

```diff
@@ -16,9 +17,11 @@ import wordCount from "../helpers/word/countWords.js";
  * @returns {WordCountResult} The number of words found in the text, plus "word" as the unit used in calculating the text length.
  */
 export default function( paper ) {
+	let text = paper.getText();
+	text = removeHtmlBlocks( text );
```

Comment:
Adding the step to remove unwanted elements in this research means that now we also filter out those elements when running this research for Word Count in the Insights collapsible. This was not the case before.

We think this is acceptable since it makes sense that we have the same result when running this research for the assessment and for Insights. Additionally, this research in the future will also be adapted to use the new HTML parser.